src/resmom/mom_main.c

```diff
@@ -9148,6 +9156,9 @@ main(int argc, char *argv[])
 	CLEAR_HEAD(task_list_timed);
 	CLEAR_HEAD(task_list_event);
 
+	cr_attrdef_idx(job_attr_def, JOB_ATR_LAST);
```

Comment:
error check?

Reply:
oops looks like i had it coded twice, removed the older one :)